client/src/util/lead_data_util.ts

```diff
@@ -30,8 +30,8 @@ export class LeadDataUtil {
    * @param prediction percent lead prediction
    */
   static formatPredictionAsLikelihood(prediction: number | null | undefined): string | null {
-    if (!prediction) return null;
-    
+    if (prediction == null) return null;
```

Comment:
Should this check for `undefined` too? Since that's a possible value according to `prediction`'s type.

Reply:
Checking for null checks both! Intellij informed me it was redundant.